src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -758,13 +762,22 @@ private static void ConfigureAuthentication(WebApplicationBuilder builder, Dashb
                     options.AddPolicy(
                         name: FrontendAuthorizationDefaults.PolicyName,
                         policy: new AuthorizationPolicyBuilder(FrontendCompositeAuthenticationDefaults.AuthenticationScheme)
-                            .RequireClaim(OtlpAuthorization.OtlpClaimName, [bool.FalseString])
+                            .RequireClaim(FrontendAuthorizationDefaults.UnsecuredClaimName)
                             .Build());
                     break;
                 default:
                     throw new NotSupportedException($"Unexpected {nameof(FrontendAuthMode)} enum member: {dashboardOptions.Frontend.AuthMode}");
             }
         });
+
+        static string ConfigureDefaultAuthScheme(DashboardOptions dashboardOptions)
```

Comment:
This deserves a comment. What downstream parts of the app rely on these schemes.